.github/workflows/integration.yml

```diff
@@ -64,3 +64,9 @@ jobs:
     uses: tarantool/mysql/.github/workflows/reusable_testing.yml@master
     with:
       artifact_name: tarantool-ubuntu-focal-${{ github.sha }}
+
+  php-client:
+    needs: tarantool
+    uses: ylobankov/client/.github/workflows/reusable_qa.yml@add-reusable-testing-workflow
```

Comment:
Change 

`ylobankov/client` -> `tarantool-php/client` 
`add-reusable-testing-workflow` -> `master` 

after https://github.com/tarantool-php/client/pull/74 is merged.

Reply:
Done